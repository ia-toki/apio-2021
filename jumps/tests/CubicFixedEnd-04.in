209636825ecd2c38299498cd26890ef71dd128c2
200 200
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 136 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 93 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200
0 0 199 199
3 79 93 93
70 88 141 141
19 158 197 197
1 12 198 198
42 82 128 128
15 46 80 80
1 37 40 40
24 177 183 183
95 178 189 189
39 76 115 115
70 142 157 157
6 19 148 148
13 86 157 157
29 75 108 108
60 186 196 196
122 198 199 199
57 105 171 171
11 55 169 169
9 16 131 131
10 14 38 38
15 74 106 106
89 193 199 199
36 57 195 195
57 157 172 172
29 89 166 166
126 161 187 187
14 145 170 170
30 116 159 159
94 113 163 163
39 134 177 177
67 175 184 184
12 43 54 54
22 120 182 182
25 174 180 180
48 59 94 94
93 175 196 196
96 190 191 191
27 64 142 142
13 140 191 191
98 165 186 186
18 80 120 120
137 140 174 174
52 149 171 171
84 104 166 166
92 165 190 190
159 182 199 199
14 35 103 103
73 94 116 116
43 189 197 197
5 14 40 40
114 148 189 189
2 45 75 75
57 103 181 181
71 148 166 166
11 17 187 187
34 54 152 152
107 123 161 161
3 5 182 182
24 68 83 83
64 143 144 144
53 81 162 162
4 146 150 150
108 139 182 182
105 161 181 181
123 135 178 178
16 184 191 191
43 47 123 123
16 76 160 160
68 190 194 194
140 194 198 198
45 186 187 187
86 169 199 199
6 82 144 144
52 69 196 196
14 124 183 183
14 16 102 102
12 13 86 86
113 116 141 141
22 133 147 147
119 120 166 166
68 78 100 100
167 168 174 174
18 45 60 60
59 79 163 163
61 142 188 188
58 82 175 175
48 56 176 176
115 157 186 186
21 79 195 195
36 95 195 195
64 118 123 123
76 115 191 191
137 154 164 164
19 185 198 198
38 141 147 147
4 89 197 197
7 48 94 94
1 8 152 152
2 4 184 184
60 122 142 142
92 125 193 193
43 159 164 164
62 155 186 186
30 107 125 125
45 47 55 55
76 182 199 199
41 148 155 155
4 119 139 139
98 152 194 194
92 139 163 163
42 50 54 54
31 115 176 176
9 11 186 186
14 127 133 133
40 41 128 128
101 135 197 197
24 73 111 111
8 36 170 170
26 27 178 178
0 10 150 150
89 114 199 199
114 114 124 124
113 121 190 190
44 55 103 103
8 12 92 92
81 151 165 165
33 163 171 171
15 64 185 185
76 80 102 102
78 78 183 183
0 15 117 117
29 125 160 160
39 43 119 119
53 72 103 103
74 95 148 148
173 194 196 196
24 38 63 63
7 28 52 52
97 143 144 144
9 12 75 75
2 107 170 170
34 134 173 173
98 109 133 133
112 169 179 179
70 104 179 179
38 40 185 185
176 178 192 192
56 81 155 155
60 123 189 189
14 37 135 135
63 111 127 127
37 44 77 77
183 183 189 189
61 181 199 199
19 62 153 153
1 6 76 76
5 13 153 153
63 119 186 186
54 118 121 121
133 147 176 176
64 129 140 140
95 142 177 177
8 9 94 94
22 48 105 105
0 99 187 187
11 18 135 135
30 85 160 160
29 73 83 83
49 108 187 187
83 169 179 179
7 13 62 62
13 52 168 168
90 104 133 133
4 7 133 133
59 94 170 170
21 159 184 184
27 77 190 190
2 3 43 43
33 45 110 110
40 126 195 195
127 133 175 175
3 56 85 85
90 102 154 154
38 90 120 120
126 188 189 189
137 181 189 189
26 42 173 173
68 86 159 159
13 58 130 130
2 4 31 31
111 113 173 173
122 127 151 151
2 2 40 40
32 176 190 190
9 31 46 46
58 75 196 196
4 4 170 170
142 154 179 179
34 51 134 134
