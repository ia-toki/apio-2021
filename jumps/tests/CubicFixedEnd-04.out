fc5005cd102fffc504bc1615010ebf88823d65f3
OK
156
-1
10
39
143
-1
34
3
6
11
-1
15
86
28
-1
10
1
36
71
-1
24
-1
6
95
15
34
26
25
24
29
42
9
11
47
6
-1
21
1
35
51
21
-1
34
22
31
25
17
-1
22
8
26
41
30
46
18
127
55
27
134
15
1
38
4
43
20
43
7
-1
41
4
4
1
30
19
84
48
-1
73
7
12
32
-1
6
15
41
46
50
77
29
73
60
5
56
10
13
6
65
-1
101
137
7
58
5
31
18
8
17
7
4
42
24
4
41
132
6
-1
62
-1
91
108
97
64
10
56
-1
80
14
8
78
-1
62
-1
25
-1
-1
13
2
25
24
1
63
35
38
24
10
44
102
14
31
54
-1
16
33
6
18
48
70
97
51
3
29
5
35
-1
-1
52
-1
32
10
52
10
49
73
29
-1
35
25
70
40
-1
60
41
29
19
-1
1
8
88
30
-1
27
39
17
38
14
15
78
123
25
-1
